tests/trainer/test_trainer.py

```diff
@@ -2512,6 +2515,7 @@ def test_accelerator_config_from_dict(self):
                     "dispatch_batches": True,
                     "even_batches": False,
                     "use_seedable_sampler": True,
+                    "gradient_accumulation_kwargs": {"sync_each_batch": True},
```

Comment:
Similarly this will only work on 0.28.0 +, which is well above the minimal version